nupic/research/frameworks/vernon/handlers.py

```diff
@@ -1278,14 +1281,17 @@ def run_task(self, task, cloned_adaptation_net):
         self.val_fast_loader.sampler.set_active_tasks(task)
 
         # Train, one batch
```

Comment:
This is actually several batches of one sample, I don't believe the "one batch" comment is still accurate.

Reply:
Good catch.